CHANGES.txt

```diff
@@ -8,6 +8,14 @@ Features
 
 * Don’t generate gallery index if the destination directory is
   site root and it would conflict with blog index (Issue #3133)
+* ``base`` and ``base-jinja`` templates now support updated
```

Comment:
Please copy the change to `bootblog4/index.tmpl` and change the line to

    * All built-in themes now support ``updated``